pkg/status/dist/templates/collector.tmpl

```diff
@@ -53,7 +53,17 @@ Collector
       {{- end }}
     {{- end}}
   {{- end }}
-{{- end }}
+
+  {{- if .Py3Warnings }}
+  Python 3 Incompatibility
```

Comment:
we should not display anything on the CLI and GUI status page for now

Reply:
+1